san/tests/test_sanbase_integration.py

```diff
@@ -73,6 +73,25 @@ def test_erc20_projects():
     assert len(result) > 0
     assert len(result[result.slug == "bitcoin"]) == 0
 
+@attr('integration')
+def test_ordinary_function():
+    queries = sanbase_graphql.QUERY_MAPPING.keys()
+
+    for query in queries:
+        if query in ["gas_used", "miners_balance", "mining_pools_distribution"]:
+            slug_to_test = "ethereum"
+        elif query in ["historical_balance", "top_holders_percent_of_total_supply", "social_dominance"]:
+            continue
+        else:
+            slug_to_test = params["project_slug"]
+        result = san.get(
+            "{}/{}".format(query, slug_to_test),
+            from_date=params["from_date"],
+            to_date=params["to_date"],
+            interval=params["interval"]
+        )
+        assert len(result.index) >=1
```

Comment:
This asserts can test a little more things. You can test that you have the expected number of returned data (if from-to is 5 days long time range and interval is "1d" you expect 5 results). Also you can test that the results have some properties (many of the metrics are non negative numbers)

Reply:
You're right, the tests should cover more criterias, not only the number of returned results. I did the new tests with a few new assertions, but will do more also.